docs/prometheus-metrics.md

```diff
@@ -69,7 +69,7 @@ rules:
 
 ### Antrea Components Scraping configuration
 Add the following jobs to Prometheus scraping configuration to enable metrics
-collection from Antrea components
+collection from Antrea components. Antrea Agent metrics endpoint is exposed through Antrea apiserver on `apiport` config parameter given in `antrea-agent.conf` (default value is 10350). Antrea Controller metrics endpoint is exposed through Antrea apiserver on `apiport` config parameter given in `antrea-controller.conf` (default value is 10349). 
```

Comment:
Made this change as there were some questions on what ports Prometheus metrics are exposed recently when integrating with Wavefront solution.